packages/core/src/components/Table/TableHeader/TableHeader.tsx

```diff
@@ -183,6 +186,7 @@ export const HvTableHeader = forwardRef<HTMLElement, HvTableHeaderProps>(
               className={classes.sortButton}
               icon
               overrideIconColors={false}
+              aria-label={sortButtonAriaLabel}
```

Comment:
can we do `sortProps`, similar to the other elements here? it's more flexible

```suggestion
              aria-label="Sort"
              {...sortProps}
```

Reply:
Yup, I'll remove `sortButtonAriaLabel` and a `sortButtonProps`

Reply:
Updated